### ISSUE TITLE
The "Too Many Requests" Freeze on Chrome need an auto Refresh to fix that

### ISSUE THREAD

**Issue by Abdoud3vil** [TheNetsky/Microsoft-Rewards-Script#147]
Hello

i noticed that after 3rd/4th account, this comes just after login, and when you refresh it 2/3 times manually, it goes and bot continue the work, happens on desktop and mobile tasks
need to add an auto refresh if this happens to fix it

![image](https://github.com/user-attachments/assets/91c8c9c6-2886-41f1-a215-13d5787af87e)

**Comment by TheNetsky** [comment]
I've never had this happen to me personally, however I only use one account for this.

**Comment by Abdoud3vil** [comment]
> I also have this problem for newly created account

those accs are created 2 years ago lel

**Comment by TheNetsky** [comment]
I guess this has something to do with reloading the 1st page to refresh the dashboard data, however I also seen people with more accounts not having this issue. This is as far as I know the first time anyone posted about this. However if you can send me the HTML you receive from this I can look into a refresh method once this page happens, since you said refreshing the page fixed this correct?

**Comment by NevilYT** [comment]
I have this problem aswell and I think good fix is if script doesn't get logs for long time it just reset and put it in config like https://github.com/TheNetsky/Microsoft-Rewards-Script/pull/118 but more updated as this were made quite long time ago and a lot have change in the script. As well potential reson for this might be that when a lot accounts going on the script doesn't close fully old chromium and you can see them in taskmanager and after time like 5 account there like 15 chromium, which might create some kind overpacking to chrome which creates this issue

**Comment by NevilYT** [comment]
but my problem is a bit different to screen shot as mine just doesn't load pages at all like forever try to load pages

**Comment by NevilYT** [comment]
> Hello
> 
> i noticed that after 3rd/4th account, this comes just after login, and when you refresh it 2/3 times manually, it goes and bot continue the work, happens on desktop and mobile tasks need to add an auto refresh if this happens to fix it
> 
> ![image](https://private-user-images.githubusercontent.com/179982912/363795594-91c8c9c6-2886-41f1-a215-13d5787af87e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU0NjQxMjYsIm5iZiI6MTcyNTQ2MzgyNiwicGF0aCI6Ii8xNzk5ODI5MTIvMzYzNzk1NTk0LTkxYzhjOWM2LTI4ODYtNDFmMS1hMjE1LTEzZDU3ODdhZjg3ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwNFQxNTMwMjZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMTIzNWMyZWE2MTEwOWExNjljNDM3MjBlNGE3YmExODQwNzU0N2JiMmY3N2U2ZmIzMDViOTRjNGExMDVhMGRhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.wJA0BoIHFdKH6IPGlH1F0VpboQjxJoNBVAz0zpqV_08)

This might be aswell because of bad proxies or overloaded ip

**Comment by serzak51** [comment]
Yeah, this error is weird. Microsoft probably added some new security.
If you are normally using a proxy, do a batch whitout a proxy. Unless your personal IP is flagged, you won't have the error, and the script will keep the cookie, so that when you start up with your proxy, you will directly be connected to your account even if the ip is sus.
